components/Downloads/ChangelogModal/index.stories.tsx

```diff
@@ -0,0 +1,431 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Button from '@/components/Common/Button';
+import ChangelogModal from '@/components/Downloads/ChangelogModal';
+import { githubProfileAvatarUrl } from '@/util/gitHubUtils';
+
+type Story = StoryObj<typeof ChangelogModal>;
+type Meta = MetaObj<typeof ChangelogModal>;
+
+const names = [
+  'ovflowd',
+  'bmuenzenmeyer',
+  'AugustinMauroy',
+  'HinataKah0',
+  'Harkunwar',
+  'rodion-arr',
+  'mikeesto',
+  'bnb',
+  'benhalverson',
+  'aymen94',
+  'shanpriyan',
+  'Wai-Dung',
+  'manishprivet',
+  'araujogui',
+  'thefossildev',
+];
+
+const Content = (
```

Comment:
what do you thinks about making a mock call `blogPostMock` ?

Reply:
I think this should be a string as I literally wrote here... https://github.com/nodejs/nodejs.org/pull/6126#issuecomment-1819592931

Reply:
I hope you read that I asked it to be a string. Otherwise this becomes a gigantic jsx constant.

Reply:
OOPS i have miss this comment